src/index.html

```diff
@@ -7,8 +7,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo">
+      </a>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="is-active nav__link">Apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
```

Comment:
```suggestion
      <a href="#home">
        <img
          src="images/logo.png"
          alt="logo"
          class="logo">
      </a>
      
      <nav class="navigation">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="is-active nav__link">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#samsung" class="nav__link">
              Samsung
            </a>
          </li>
```
fix your code style all over the page